src/comments/entities/comment.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BlogEntity } from '../../blogs/model/blog.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Tree,
+  UpdateDateColumn,
```

Comment:
You are importing `Tree` but you are not using it !
Here is doc link about it with few examples: 
https://typeorm.io/#/tree-entities/closure-table